doc/UsersGuide/source/fmitlm.rst

```diff
@@ -91,50 +91,53 @@ SUNDIALS manualy and create a build script as well.
 FMI Import
 ~~~~~~~~~~
 
-To import the FMU package use the OpenModelica command importFMU,
+If you want to simulate a single, stand-alone FMU, or possibly a connection
+of several FMUs, the recommended tool to do that is OMSimulator, see the
+`OMSimulator documentation <https://openmodelica.org/doc/OMSimulator/master/html/>`_
+for further information.
+
+FMI Import allows to use an FMU, generated according to the FMI for Model
+Exchange 2.0 standard, as a component in a Modelica model. This can be
+useful if the FMU describes the behaviour of a component or sub-system in a
+structured Modelica model, which is not easily turned into a pure FMI-based
+model that can be handled by OMSimulator.
+
+FMI is a computational description of a dynamic model, while a Modelica model is
+a declarative description; this means that not all conceivable FMUs can be successfully
+imported as Modelica models. Also, the current implementation of FMU import in
+OpenModelica is still somewhat experimental and not guaranteed to work in all
+cases. However, if the FMU-ME you want to import was exported from a Modelica model
+and only represents continuous time dynamic behaviour, it should work without problems
+when imported as a Modelica block.
+
+Please also note that the current implementation of FMI Import in OpenModelica
+is based on a built-in wrapper that uses a `reinit()` statement in an algorithm
+section. This is not allowed by the Modelica Language Specification, so it is
+necessary to set the compiler to accept this non-standard construct by setting
+the `--allowNonStandardModelica=reinitInAlgorithms` compiler flag. In OMEdit,
+you can set this option by activating the *Enable FMU Import* checkbox in the
+*Simulation Setup | Translation Flags* tab. This will generate a warning during
```

Comment:
Its better to write,

"In OMEdit, you can set this option by activating the *Enable FMU Import* checkbox in the
*Tools | Options | Simulation | Translation Flags*."

Of course one can overwrite the option in `Simulation Setup`.

Reply:
OK